public/app/features/alerting/unified/components/AlertLabels.tsx

```diff
@@ -28,6 +29,7 @@ export const AlertLabels = ({ labels, commonLabels = {}, size }: Props) => {
 
   const commonLabelsCount = Object.keys(commonLabels).length;
   const hasCommonLabels = commonLabelsCount > 0;
+  const tooltip = t('alert-labels.button.show.tooltip', 'Show common labels');
```

Comment:
changes in this file are because are not related to the pr, because was a change i reverted (adding a limit to these props). And betterer forced me to add translations.